### PR TITLE
Add terminal output for friendly feedback

### PR DESCRIPTION
**Pull request by wanieldilson**
# What

Adding some output in the terminal for users running the script.

# Why

For some instant feedback.

# Screenshots
image

# Notes

### CODE REVIEW

**Comment by Gary-H9** [comment]
LGTM

**Review by Gary-H9 — APPROVED** [review]
LGTM

File: scripts/config-parser/bin/dhcp_config_parser.rb [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 require_relative "../lib/dhcp_config_parser"
 
+puts "There are #{DhcpConfigParser.run.length} differences in reservation data"
```
Comment (richrace) [inline scripts/config-parser/bin/dhcp_config_parser.rb:5]: I would store `DhcpConfigParser.run` into a variable so it only has to run once.

**Review by richrace — APPROVED** [review]
👍

**Pull request merged**